### PR TITLE
Build on Linux (Ubuntu) (RUNNING.md)

### PR DESCRIPTION
**Pull request by olivia1246**
Added some missing possibly useful information.

Information taken from here: https://github.com/devos50/qemu-ios/issues/92

### CODE REVIEW

**Review by MrDiagnose — CHANGES_REQUESTED** [review]
it will be `./configure` after entering `qemu-ios` directory

**Comment by olivia1246** [comment]
> it will be `./configure` after entering `qemu-ios` directory

Oops I will fix that. Thank you

File: RUNNING.md [hunk 1]
```diff
@@ -36,16 +36,16 @@ Compile QEMU by running the following commands from the root directory:
 mkdir build
 cd build
 # On Intel Macs
-../configure --target-list=arm-softmmu --extra-cflags=-I/usr/local/opt/openssl@3/include --extra-ldflags='-L/usr/local/opt/openssl@3/lib -lcrypto'
+./configure --target-list=arm-softmmu --extra-cflags=-I/usr/local/opt/openssl@3/include --extra-ldflags='-L/usr/local/opt/openssl@3/lib -lcrypto'
```
Comment (Florin9doi) [inline RUNNING.md:40]: no, the `build` dir is empty , it is created 2 lines above

Comment (MrDiagnose) [reply RUNNING.md:40]:
```
mkdir build
cd build
```
can be removed when using `./configure` since build directory is created automatically

Comment (Florin9doi) [reply RUNNING.md:40]: Shouldn't rely on legacy and undocumented mechanisms, the official qemu documentation recommends to create the `build` folder manually: https://www.qemu.org/docs/master/devel/build-system.html

Comment (MrDiagnose) [reply RUNNING.md:40]: ok 👌

Comment (devos50) [reply RUNNING.md:40]: Agreed, I think it is standard to create the `build` directory yourself. Can you adjust the PR accordingly?

Comment (olivia1246) [reply RUNNING.md:40]: Okay I will modify it thank you for the feedback!

**Review by Florin9doi — COMMENTED** [review]
https://github.com/devos50/qemu-ios/pull/143/commits/1c6469394bc22a99dcb40c37c60e811757529ac9 is empty

**Review by devos50 — APPROVED** [review]

**Pull request merged**